go/logger/interfaces.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package logger
+
+import (
+	"golang.org/x/net/context"
+)
+
+type FLogger interface {
+	CriticalfLogger
+	DebugfLogger
+	ErrorfLogger
+	FatalfLogger
+	InfofLogger
+	NoticefLogger
+	WarningfLogger
+}
+
+type ContextFLogger interface {
+	CCriticalfLogger
```

Comment:
I was thinking these single method interfaces could help with composing other interfaces, even outside package. So for example, when you need a logger that can do `Debugf` and `Fatalf`, instead of listing out those methods signatures in your local interface in the package, like:
```
type theLoggerINeed interface {
    Debugf(s string, v ...interface{})
    Fatalf(s string, v ...interface{})
}
```

, you could simply do

```
type theLoggerINeed interface {
    logger.DebugfLogger
    logger.FatalfLogger
}
```

Thanks for the reminder on comments!